### PR TITLE
Fix crash issue after add operation implement.

### PR DESCRIPTION
**Pull request by lisa0314**
Fix [intel/web-polyfill#396](https://github.com/intel/webml-polyfill/issues/396)
@fujunwei Please help to review.

### CODE REVIEW

File: services/ml/compilation_impl_mac_bnns.mm [hunk 1]
```diff
@@ -61,10 +61,24 @@ bool CompileAdd(OperationMac& operation,
...
+    DLOG(ERROR) << "Broadcasting is not supported by now!";
+    return false;
+  } else {
+    for (size_t i = 0; i < input_a.dimensions.size(); ++i) {
+      if (input_a.dimensions[i] != input_b.dimensions[i]) {
+        DLOG(ERROR) << "Broadcasting is not supported by now!";
+        return false;
+      }
+    }
+  }
```
Comment (fujunwei) [inline services/ml/compilation_impl_mac_bnns.mm]: Maybe [operator==](https://en.cppreference.com/w/cpp/container/vector/operator_cmp) of std::vector can satisfy your requirement.

Comment (lisa0314) [reply services/ml/compilation_impl_mac_bnns.mm]: Yes.

**Comment by lisa0314** [comment]
@fujunwei Please help to merge it.

**Comment by fujunwei** [comment]
LGTM, thanks.

**Pull request merged**